src/main/resources/twilio-node/operation.mustache

```diff
@@ -0,0 +1,61 @@
+{{#hasParams}}
+    {{#hasRequiredParams}}
+        if (params === null || params === undefined) {
+            throw new Error('Required parameter "params" missing.');
+        }
+
+        {{#requiredParams}}
+        {{^isBodyParam}}
```

Comment:
Where are we passing this parameter(isBodyParam) to the mustache file?

Reply:
This is done by the base openapi-generator which is populating the properties. No customization is needed.